benchmarking/commons/benchmark_definitions/common.py

```diff
@@ -11,11 +11,12 @@
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
 from dataclasses import dataclass
+from pathlib import Path
 from typing import Optional, List
 
 
 @dataclass
-class BenchmarkDefinition:
+class SurrogateBenchmarkDefinition:
```

Comment:
Maybe we can add some documentation here, explaining the difference between real and surrogate benchmark?

Reply:
OK